### DISCUSSION TITLE
Welcome to WinP Discussions!

### DISCUSSION THREAD

**Discussion by SL1dee36** [SL1dee36/WinP#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ghostly-DeeF** [comment]
Гоооол

**Comment by Sskyindiffference** [comment]
Cool, could you add popup notifications when archiving and an interface instead of the console when using the context menu?

**Comment by slide36** [comment]
❤️
